content/posts/js/webpack/index.md

```diff
@@ -0,0 +1,405 @@
+---
+date: "2015-06-16"
+title: Introduction à Webpack
+tags:
+  - javascript
+  - task-runner
+  -
+authors:
+  - MoOx
+header:
+  linearGradient: 160deg, rgb(204, 51, 51), rgba(204, 51, 51, .6)
+  credit: https://www.flickr.com/photos/luciano_meirelles/3461046001/
+---
+
+# Pourquoi webpack ?
+
+Avant webpack, nous avions beau avoir des super task runners comme
+[grunt](/posts/js/premiers-pas-avec-grunt)
+ou
+[gulp](/posts/js/introduction-gulp),
+il faut avouer qu'on bricolait quand même pas mal.
+
+Pourquoi ? Devoir ajuster des configurations/définition de tâches pour ajuster
+les copies des images, fonts... Ajuster les urls de générations...
+Il reste avec ces solutions une grande partie de rafistolage, où l'on se doit de
+faire très attention à la moindre réorganisation de code, sous peine de tout
+casser une partie du rendu.
+
+Prenons par exemple une image de fond déclaré en CSS. Qui vous préviendra qu'une
```

Comment:
> Do not review typo yet please, otherwise feel free to review the content ^^.
> https://github.com/putaindecode/putaindecode.fr/pull/465#issue-90371020